datafusion/expr/src/planner.rs

```diff
@@ -133,6 +133,17 @@ pub trait UserDefinedSQLPlanner: Send + Sync {
     fn plan_extract(&self, args: Vec<Expr>) -> Result<PlannerResult<Vec<Expr>>> {
         Ok(PlannerResult::Original(args))
     }
+
+    fn plan_create_struct(&self, args: Vec<Expr>) -> Result<PlannerResult<Vec<Expr>>> {
```

Comment:
Could we please document these functions on the traits 

I think it would be especially useful to document what the arguments meant, and how the arguments between create_struct and create_named_struct differ

Perhaps we can lift the documentation from https://datafusion.apache.org/user-guide/sql/scalar_functions.html#struct and https://datafusion.apache.org/user-guide/sql/scalar_functions.html#named-struct

Reply:
Sure. I will add detailed documentation